doc/rst/usingchapel/prereqs.rst

```diff
@@ -18,29 +18,34 @@ about your environment for using Chapel:
   * You are using an environment that supports standard UNIX commands
     such as: ``cd, mkdir, rm, echo``
 
-  * You have the Bourne shell available at ``/bin/sh``, the C-shell
+  * You have a Bourne shell available at ``/bin/sh``, the C-shell
```

Comment:
Should it be "a C-shell" as well?

Reply:
Oh, we don't need the csh anymore. I'll remove that part.